misc/testerina/modules/testerina-core/src/main/ballerina/Module.md

```diff
@@ -23,8 +23,8 @@ function beforeFunc() {
 
 // Test function
 @test:Config {
-    before: "beforeFunc",
```

Comment:
Since we say that it's a simple test suite, shall we remove all these annotations and attributes and make it the simplest test suite with only one test case? 
Then, may be we can add and describe the following capabilities.

1. Setup and tear down pre-requisites and post actions for a test suite, test group, per each test or for a single test. We can include a sample for beforeSuite and afterSuite and mention that similarly this can be achieved for other cases as well.

2. Provide data sets to run the tests against using data provider

3. Ensure test execution order using 'dependsOn'

WDYT?

Reply:
This https://github.com/ballerina-platform/ballerina-distribution/blob/master/doc-guidelines.md#ballerina-api-documentation-guidelines says,

> Module-level documentation page needs to introduce the general concept of the API and the main APIs that are used.